src/components/MajorFilter.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Autocomplete,
+  Popover,
+  Chip,
+  OutlinedInput,
+  IconButton,
+} from '@mui/material';
+import { MAJORS } from '../types/options';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import ArrowDropUpIcon from '@mui/icons-material/ArrowDropUp';
+
+function MajorFilter() {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [majors, setMajors] = useState([]);
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
How about initializing this to true?

Reply:
Or we can we make it such that `isOpen` changes when the user starts typing.

Reply:
I added `onInputChange` back.